File: apps/institution-api/Models/Dto/PsiSpecializationCountByCategoryDto.cs
namespace ProviderApi.Models.Dto
{
  public class PsiSpecializationCountByCategoryDto
  {
    public string CipSubSeriesCode { get; set; }
    public string CipSubSeries { get; set; }
    public int? ProgramCount { get; set; }
    public int? SpecailizationCount { get; set; }
    public int? SessionCount { get; set; }
    public int? ProviderCount { get; set; }
  }
}

File: libs/common/models/psi-specialization-count-by-category-dto.ts



export class PsiSpecializationCountByCategoryDto {
    public cipSubSeriesCode: string = '';
    public cipSubSeries: string = '';
    public programCount: number;
    public specailizationCount: number;
    public sessionCount: number;
    public providerCount: number;

    constructor(init? : Partial<PsiSpecializationCountByCategoryDto>) {
        
        Object.assign(this, init);
    }
}

